src/main/java/com/crowdin/cli/commands/picocli/DownloadSubcommand.java

```diff
@@ -55,9 +55,13 @@ class DownloadSubcommand extends ActCommandWithFiles {
 
     @Override
     protected NewAction<PropertiesWithFiles, ProjectClient> getAction(Actions actions) {
-        return (dryrun)
-            ? actions.listTranslations(noProgress, treeView, false, plainView, all, true)
-            : actions.download(new FsFiles(), noProgress, languageId, pseudo, branchName, ignoreMatch, isVerbose, plainView, all);
+        if (dryrun) {
```

Comment:
this part of the code shouldn't be affected, the only thing we need - just to additionally check the "skip_untranslated_files" presence in case there are no files in the downloaded archive